libs/kotaemon/kotaemon/indices/ingests/extensions.py

```diff
@@ -0,0 +1,140 @@
+from copy import deepcopy
+from typing import Any
+
+from decouple import config
+from llama_index.core.readers.base import BaseReader
+from theflow.settings import settings as flowsettings
+
+from kotaemon.loaders import (
+    AdobeReader,
+    AzureAIDocumentIntelligenceLoader,
+    GOCR2ImageReader,
+    HtmlReader,
+    MhtmlReader,
+    PandasExcelReader,
+    PDFThumbnailReader,
+    TxtReader,
+    UnstructuredReader,
+)
+
+unstructured = UnstructuredReader()
+adobe_reader = AdobeReader()

```

Comment:
@phv2312 let add AdobeReader and AzureDIReader as selectable options on Extensions UI. Also remove it from Retrieval settings since it is no longer needed :D